pandas_datareader/data.py

```diff
@@ -698,8 +698,19 @@ def _option_frames_from_url(self, url):
 
     def _get_underlying_price(self, url):
         root = self._parse_url(url)
-        underlying_price = float(root.xpath('.//*[@class="time_rtq_ticker Fz-30 Fw-b"]')[0]\
-            .getchildren()[0].text)
+        underlying_price = root.xpath('.//*[@class="time_rtq_ticker Fz-30 Fw-b"]')[0]\
+            .getchildren()[0].text
+
+        try:
+            underlying_price = float(underlying_price)
+        except ValueError:
+            # check for comma
+            underlying_price = underlying_price.replace(',', '')
+
+        try:
```

Comment:
Note: You could probably indent this block so float is only applied once is the usual case, or always replace (which may be cleaner).